test/unit/org/apache/cassandra/service/accord/AccordTestUtils.java

```diff
@@ -420,7 +401,12 @@ public static AccordCommandStore createAccordCommandStore(
                                                            time,
                                                            new AccordAgent(),
                                                            null,
-                                                           cs -> NOOP_PROGRESS_LOG,
+                                                           cs -> new NoOpProgressLog(),
+                                                           cs -> new DefaultLocalListeners(new RemoteListeners.NoOpRemoteListeners(), new DefaultLocalListeners.NotifySink()
```

Comment:
Are no op listeners going to be sufficient for all the tests that use this?

Reply:
None of the tests break!